src/editor/tests.py

```diff
@@ -242,125 +242,247 @@ def test_editor_review_round_cancel(self):
+                'editor_review',
+                kwargs={
+                    'submission_id': self.book.id
+                }
+            ),
+            {'new_round': ''}
+        )
+
         resp = self.client.get(
-            reverse('editor_review', kwargs={'submission_id': self.book.id}))
+            reverse(
+                'editor_review',
+                kwargs={
+                    'submission_id': self.book.id
+                }
+            )
+        )
         content = resp.content
         self.assertEqual(resp.status_code, 200)
         self.assertEqual("403" in content, False)
         self.assertEqual("btn-task" in content, True)
-        resp = self.client.get(reverse('editor_review_round',
-                                       kwargs={'round_number': 1,
-                                               'submission_id': self.book.id}))
+
+        resp = self.client.get(
+            reverse(
+                'editor_review_round',
+                kwargs={
+                    'round_number': 1,
+                    'submission_id': self.book.id
+                }
+            )
+        )
         content = resp.content
         self.assertEqual(resp.status_code, 200)
         self.assertEqual("403" in content, False)
         self.assertEqual("btn-task" in content, True)
         self.assertEqual("ROUND 1" in content, True)
         review_file = tempfile.NamedTemporaryFile(delete=False)
-        resp = self.client.get(reverse('editor_add_reviewers',
-                                       kwargs={'round_number': 1,
-                                               'submission_id': self.book.id,
-                                               'review_type': 'internal'}))
+
+        resp = self.client.get(
+            reverse(
+                'editor_add_reviewers',
+                kwargs={
+                    'round_number': 1,
+                    'submission_id': self.book.id,
+                    'review_type': 'internal'
+                }
+            )
+        )
         content = resp.content
         self.assertEqual(resp.status_code, 200)
         self.assertEqual("403" in content, False)
-        resp = self.client.post(reverse('editor_add_reviewers',
-                                        kwargs={'round_number': 1,
-                                                'submission_id': self.book.id,
-                                                'review_type': 'internal'}),
-                                {'message': 'Dear reviewer',
-                                 'due_date': '2015-11-11',
-                                 'review_form': 'rua_test_form', 'committee': 2,
-                                 'reviewer': 4, 'attachment': review_file})
+
+        # Delete all review assignments
+        # The below request creates a duplicate
+        # of an assignment described in the fixture test/test_core_data
+        for review_assignment in core_models.ReviewAssignment.objects.all():
```

Comment:
This seems potentially dangerous. Is there any risk of running this test on a live instance of RUA?

Reply:
Django tests use a special testing database, never the one defined in the project's settings. If I'm not mistaken the state of this database is reset for each individual test.